src/Loop.php

```diff
@@ -59,155 +59,162 @@ public static function stop()
     /**
      * Defer the execution of a callback.
      *
-     * @param callable $callback The callback to defer.
+     * @param callable(mixed $data, string $watcherIdentifier) $callback The callback to defer.
```

Comment:
Possibly, it's a thing which always had annoyed me … so maybe we could fix it in the standard ;-)

Also, typically you could also just store the watcher via `$watcher = Loop::defer($cb, [&$watcher])` into $cbData. _If_ (which you don't really) need it, you always can do it.